StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/New Payment Method Screen/WalletHeaderView.swift

```diff
@@ -63,11 +63,12 @@ extension PaymentSheetViewController {
 
         private lazy var payWithLinkButton: PayWithLinkButton = {
             let button = PayWithLinkButton()
-            if LiquidGlassDetector.isEnabled {
+            if appearance.cornerRadius == nil, LiquidGlassDetector.isEnabledInMerchantApp {
                 button.ios26_applyCapsuleCornerConfiguration()
             } else {
-                button.cornerRadius = appearance.cornerRadius
+                button.cornerRadius = appearance.cornerRadius ?? 6
```

Comment:
Should we have some centralized place where we have logic that does `appearance.cornerRadius ?? 6`?

Reply:
Initially I was thinking we do something like this:
```
internal extension PaymentSheet.Appearance {
    var cornerRadiusOrDefault: CGFloat {
        cornerRadius ?? 6.0
    }
}
```
but, i think the intent is less clear.  `static let defaultPreiOS26CornerRadius: CGFloat = 6` is at least better.